livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java

```diff
@@ -83,9 +82,16 @@ public Integer createSession(InterpreterContext context, String kind) throws Exc
           }.getType());
       Integer sessionId = ((Double) jsonMap.get("id")).intValue();
       if (!jsonMap.get("state").equals("idle")) {
-        Integer nosRetry = MAX_NOS_RETRY;
+        Integer retryCount = 120;
```

Comment:
@felixcheung what would be the ideal default retry count here? can we revert it to 60 ?